lang/po/Firmware.pot

```diff
@@ -2460,3 +2460,7 @@ msgstr ""
 #. MSG_FW_MK3_DETECTED c=20 r=4
 msgid "MK3 firmware detected on MK3S printer"
 msgstr ""
+
+#. MSG_REPRINT c=7 
```

Comment:
Submenu can be `c=18` long

Reply:
`.pot` files and `.po`files have to be updated with `lang/update-pot.py` and `lang/update-po-py`.